src/purva/ScannerClass/SwitchcaseLadderAssignment_15/BankOperation.java

```diff
@@ -0,0 +1,134 @@
+package purva.ScannerClass.SwitchcaseLadderAssignment_15;
+
+public class BankOperation {
+
+	int iciciBalance =5000;
+	int hdfcBalance = 9000;
+	int bobBalance =8000;
+	int sbiBalance =700;
+	
+	int debitCounticici=0,debitCounthdfc=0,debitCountbob=0,debitCountsbi=0,creditCounticici=0,creditCounthdfc=0,creditCountbob=0,creditCountsbi=0;
+	static int totalDebitCount=0, totalCreditCount=0;
+	void operation(String bankName,String transactionType,int amount) {
+		switch(transactionType) {
+		case "Debit":
+			switch(bankName) {
+			case "ICICI":
+				if(iciciBalance>=amount) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method